base_layer/p2p/src/config.rs

```diff
@@ -95,8 +95,6 @@ pub struct P2pConfig {
     /// The maximum number of concurrent outbound tasks allowed before back-pressure is applied to outbound messaging
     /// queue
     pub max_concurrent_outbound_tasks: usize,
-    /// The size of the buffer (channel) which holds pending outbound message requests
-    pub outbound_buffer_size: usize,
```

Comment:
This reduction in config is nice, as all the `outbound_buffer_size` in the config files were a bit confusing.